src/ActionRunner/Resources.resx

```diff
@@ -59,7 +59,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="DOTNET_CORE_DOWNLOAD_FAILURE" xml:space="preserve">
-    <value>Couldn't download .NET Core Desktop Runtime 3.1, please install it manually.</value>
```

Comment:
I know these are minor fixes, but please split stuff like this into separate PRs, otherwise PR/commit name "Update Windows Implementation Library" is misleading.

Reply:
This one actually needs to stay because it will not compile